matkono/HealthSup
# Feature/refactoring services

## What this PR does

- Remove business logic to service domain

## How to check if this PR is correct

- Ensure all scenarios.

- Authenticate failed:

![image](https://user-images.githubusercontent.com/12301754/82764886-b998e500-9de8-11ea-90f1-2252ae34840c.png)

- Authenticate success:

![image](https://user-images.githubusercontent.com/12301754/82764892-c1f12000-9de8-11ea-96a7-6859c8d2b488.png)

- Unauthorized:

![image](https://user-images.githubusercontent.com/12301754/82764900-d0d7d280-9de8-11ea-9eac-7e039ee88598.png)

- Unauthorized when token expires:

![image](https://user-images.githubusercontent.com/12301754/82764911-e2b97580-9de8-11ea-9309-f3f2d4d429bf.png)

- Login failed:

![image](https://user-images.githubusercontent.com/12301754/82764914-ebaa4700-9de8-11ea-9232-b8e50a1e8e7e.png)

- Login success:

![image](https://user-images.githubusercontent.com/12301754/82764921-f238be80-9de8-11ea-9061-9fc111205aa6.png)

- Update password failed:

![image](https://user-images.githubusercontent.com/12301754/82764924-fb299000-9de8-11ea-8731-f262d15e736f.png)

- Update password success:

![image](https://user-images.githubusercontent.com/12301754/82764930-041a6180-9de9-11ea-87d4-009d1fcc494e.png)
